core/rawdb/database.go

```diff
@@ -355,6 +356,8 @@ func InspectDatabase(db ethdb.Database, keyPrefix, keyStart []byte) error {
 			storageSnaps.Add(size)
 		case bytes.HasPrefix(key, preimagePrefix) && len(key) == (len(preimagePrefix)+common.HashLength):
 			preimages.Add(size)
+		case bytes.HasPrefix(key, configPrefix) && len(key) == (len(configPrefix)+common.HashLength):
```

Comment:
Can we just count the chain config into the `meta data`? It doesn't have too much meaning to have a standalone type.

Reply:
I think we also need to merge the `Shutdown metadata` into the `metadata` category. @holiman what's your opinion?

Reply:
Yes, I agree